e2e/testinfra/terraform/modules/service_account/service_account.tf

```diff
@@ -35,6 +35,12 @@ resource "google_service_account_iam_member" "k8s_sa_ns_binding" {
   member             = "serviceAccount:${data.google_project.project.project_id}.svc.id.goog[config-management-system/ns-reconciler-test-ns]"
 }
 
+resource "google_service_account_iam_member" "k8s_sa_monitoring_binding" {
+  service_account_id = google_service_account.gcp_sa.name
+  role               = "roles/iam.workloadIdentityUser"
+  member             = "serviceAccount:${data.google_project.project.project_id}.svc.id.goog[config-management-monitoring/default]"
```

Comment:
This is a different principal than the RootSync/RepoSync reconciler KSA. I don't think the intent is to create a binding for this principal to the other GSAs that use this module.

For this use case it might make sense to create a separate module or just create the binding directly since this is a one-off. The reusable module was defined for the other bindings since we needed to create multiple different bindings for the same KSA.

Reply:
A one-off setup should be good enough, can it be moved to common/terraform.tf?

Reply:
`common/service_accounts.tf` would be a good place imo

Reply:
Changed to a one-off setup in `common/service_accounts.tf`

Reply:
Ah wait the role was left out

Reply:
Added role.